client/store/index.js

```diff
@@ -23,84 +23,66 @@ import Vue from 'vue';
 import Vuex from 'vuex';
 import VuexPersistence from 'vuex-persist';
 import {
+  // graph
+  getGraphDefaultState,
+  graphGetters,
+  graphMutations,
+
+  // route
+  routeActionCreator,
+  routeGetters,
+
+  // settings
   getSettingsWorkflowHistoryDefaultState,
-  getWorkflowHistoryDefaultState,
   settingsWorkflowHistoryGetters,
   settingsWorkflowHistoryMutations,
-} from '~containers';
 
-// Graph store
+  // workflow history
+  getWorkflowHistoryDefaultState,
+} from '~containers';
 
-const getGraphDefaultState = () => ({
-  childRoute: null,
-  newExecutionId: null,
-  parentRoute: null,
-  hasChildBtn: false,
-  childBtnText: null,
-  parentBtnText: 'to parent',
+const getDefaultState = (state = {}) => ({
```

Comment:
how the default state looks like? is it something related to a cadence workflow state?

Reply:
for the vuex store it will take all of the sub-store states (i.e. graph, route, workflow, workflowHistory etc.) and combine into one store which can be queried or mutated via getters or mutations. A container has access to its sub-store, but it can also query or mutate other stores if necessary.

One example could be on the workflow pending screen (in a future PR) where clicking on the filters will update the current filter using the ROUTE_UPDATE_QUERY action, even though its in a separate store to workflowPending.